portal-api/src/v1/users/user.service.test.js

```diff
@@ -0,0 +1,83 @@
+const { produce } = require('immer');
+const { cloneDeep } = require('lodash');
+const { UserService } = require('./user.service');
+const { TEST_USER_TRANSFORMED_FROM_DATABASE } = require('../../../test-helpers/unit-test-mocks/mock-user');
+const { STATUS } = require('../../constants/user');
+const UserBlockedError = require('../errors/user-blocked.error');
+const UserDisabledError = require('../errors/user-disabled.error');
+
+describe('UserService', () => {
+  let userService;
+  let testUser;
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    userService = new UserService();
+    testUser = cloneDeep(TEST_USER_TRANSFORMED_FROM_DATABASE);
+  });
+
+  describe('when the user is blocked', () => {
+    let blockedUser;
+    beforeEach(() => {
+      blockedUser = produce(testUser, (draft) => {
+        draft['user-status'] = STATUS.BLOCKED;
+      });
+    });
+
+    it('validating user throws a UserBlockedError', async () => {
```

Comment:
Do we need an assertion where `disabled` property does not exists and `user-status: active`?